EasyModbusSource/src/test/java/io/vantiq/extsrc/EasyModbusSource/TestEasyModbusBase.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2020 Vantiq, Inc.
+ *
+ * All rights reserved.
+ * 
+ * SPDX: MIT
+ */
+
+package io.vantiq.extsrc.EasyModbusSource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import org.junit.BeforeClass;
+
+public class TestEasyModbusBase {
+    static String testIPAddress;
+    static int testIPPort;
+    static int testSize;
+    static String testVantiqServer;
+    static String testAuthToken;
+    static String testSourceName;
+    static String testTopicName;
+    static String testTypeName;
+    static String testProcedureName;
+    static String testRuleName;
+
+    @BeforeClass
+    public static void getProps() {
+        testIPAddress = System.getProperty("EntConIPAddress", "127.0.0.1");
```

Comment:
Generally, you probably don't want to provide this value as a default.  The `assumeTrue()` calls will abort (skip) the test if that value's not there.  This guarantees that the value is always there.

The value should only come from the user's gradle.properties file or command-line flags.

Reply:
bump